.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
+v                               ✰  Thanks for creating a PR! ✰    
+v    Before smashing the submit button please review the checkboxes.
+v    If a checkbox is n/a - please still include it but + a little note why
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/tendermint/tendermint/blob/master/CONTRIBUTING.md))
+
+- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
+- [ ] Wrote tests
+- [ ] Updated relevant documentation (`docs/`)
+- [ ] Added a relevant changelog entry: `clog add [section] [stanza] [message]`
+- [ ] rereviewed `Files changed` in the github PR explorer
```

Comment:
Should we still add a line here that states sth like:
```suggestion
- [ ] rereviewed `Files changed` in the github PR explorer
- [ ] bumped version number of crate if this is meant to be released
- [ ] bumped supported version in README.md if tendermint version was updated
```
Also not sure if "rereviewed" shouldn't be just "reviewed"?

Reply:
does it still make sense to bump version in PR now, since we are doing master and develop branches?

Reply:
No, you are right.